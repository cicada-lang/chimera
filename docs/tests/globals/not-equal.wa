print find x {
  NotEqual(x, 1)
}
