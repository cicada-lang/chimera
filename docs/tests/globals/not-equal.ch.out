[_.0 with { NotEqual(_.0, 1) }]
